scripts/test

```diff
@@ -80,10 +80,7 @@ run_rancher()
     while sleep 2; do
```

Comment:
Added the output logs akin to the provisioning tests. Maybe we combine the logic there someday, but for now this should suffice.